app/core/src/test/java/com/fsck/k9/message/html/UriMatcherTest.kt

```diff
@@ -82,6 +83,44 @@ class UriMatcherTest {
         )
     }
 
+    @Test
+    fun textIsValidHttpUri() {
+        assertPastedUriFound(
+            "https://github.com",
+        )
+    }
+
+    @Test
+    fun textIsValidMailToUri() {
+        assertPastedUriFound(
+            "mailto:mail@email.com",
+        )
+    }
+
+    @Test
+    fun invalidHttpUriWithSpaceInUri() {
+        assertPastedUriNotFound(
+            "https:// github.com",
+        )
+    }
+
+    @Test
+    fun invalidMailToUriPrefixedWithSpace() {
+        assertPastedUriNotFound(
+            " mailto:mail@email.com",
+        )
+    }
+
+    private fun assertPastedUriFound(text: String) {
+        val result = UriMatcher.isValidUri(text)
+        assert(result)
+    }
+
+    private fun assertPastedUriNotFound(text: String) {
```

Comment:
Please rename this to not contain the word "pasted", e.g. to assertValidUriNotFound.